tgui/packages/tgui/interfaces/SlimePenController.js

```diff
@@ -214,35 +214,28 @@ const StoreViewer = (_, context) => {
       <Section textAlign="center" fontSize="18px">
         Corral Data
       </Section>
-      <LabeledList>
-        <LabeledList.Item label="Corral Upgrades">
-          <Collapsible>
-            {corral_upgrades.map((item) => (
-              <Section
-                style={{ 'border-radius': '5px' }}
-                mb="6px"
-                key={item.name}>
-                <Stack fill>
-                  <Stack.Item>
-                    <Box fontSize="16px">{item.name}</Box>
-                  </Stack.Item>
-                  <Stack.Item>
-                    <Button
-                      ml="10px"
-                      mt="3px"
-                      style={{
-                        'transform': 'scale(1.5)',
-                      }}
-                      icon={'question'}
-                      tooltip={item.desc}
-                    />
-                  </Stack.Item>
-                </Stack>
-              </Section>
-            ))}
-          </Collapsible>
-        </LabeledList.Item>
-      </LabeledList>
+      <Collapsible title="Active Corral Upgrades">
```

Comment:
~~guh can't we just use typescript for this~~

Reply:
if i knew how to write it, sure
youre free to do it, but if it works it works